src/SourceBuild/tarball/content/prep.sh

```diff
@@ -79,13 +79,13 @@ while read -r line; do
     if [[ $line == *"Private.SourceBuilt.Artifacts"* ]]; then
         if [ "$downloadArtifacts" == "true" ]; then
             echo "  Downloading source-built artifacts from $line..."
-            (cd $SCRIPT_ROOT/packages/archive/ && curl -O $line)
+            (cd $SCRIPT_ROOT/packages/archive/ && curl --retry 5 -O $line)
```

Comment:
I wasn't able to find anything concrete - I looked up SSL errno 110 and it seemed like it might just be a general unknown network error. I didn't find any official documentation on it. So yes, this is more of a try-and-see situation.